content/project-news/reactos-0414-released.md

```diff
@@ -0,0 +1,117 @@
+
+Let's not forget the PnP (Plug n' Play) manager of the kernel which is also an important part not only in regard to stability, but also the usability of the system with a plethora of hardware.
+Vadim Galyant added the initial headers implementation of the Resources Arbiter library which can be used by the bus
+drivers like ACPI, PCI or legacy PIC HAL. In addition to that Vadim implemented debug code both in PnP and I/O managers whereas Eric Kohl improved the PnP manager to map device capabilities to capability flags. Thomas Faber fixed a critical bug that could
+cause memory corruption in the kernel space and fixed a bad IRP (I/O request packet) handling that caused a [Blue Screen](https://en.wikipedia.org/wiki/Blue_screen_of_death) on an Original Xbox console with USB enabled. 
+
+The Kernel Debugger (KD) and related modules have also received updates from developers and contributors alike. Dmitry Borisov added ComPort library for NEC PC-98 which is fundamental in the further development and support of NEC series. Hermès Bélusca-Maïto has finished the support for debug filters, improved the `cregs` and `tss` commands included in the interactive kernel debugger (KDBG, present only in GCC debug builds),
+rewrote the TSS handling code and squashed various bugs with it. Namely the issue with the KDBG backtrace when a TSS switch happened during execution. Furthermore, Jérôme Gardou alongside with Hervé Poussineau brought some fixes and improvements to the KDBG module.
+
+## NetKVM VirtIO bringup
+NetKVM VirtIO is an NDIS 5 compatible driver developed by Red Hat Inc., which allows fast VirtIO networking in virtual machine software such as QEMU and other KVM hardware accelerated virtual machines.
+Nguyen Trung Khanh, a ReactOS contributor, took over Benjamin Aerni's work of importing the driver into the source tree of ReactOS.
+This provides ReactOS with support for another network interface out of the box.
+
+## Miscellaneous changes & improvements
+Other areas of ReactOS infrastructure have received lots improvements by various contributors, ranging from device drivers to software applications. Notably FreeLoader, the ReactOS' bootloader, received incremental fixes and improvements. As such Hermès Bélusca-Maïto added support for booting Linux 64-bit systems in 64-bit FreeLoader and fixed an issue where
+FreeLoader couldn't read from an EXT2 volume hence preventing booting. Dmitry Borisov fixed a serious triple fault bug when ReactOS was booted up in Screen debugging mode. In addition to that, Dmitry Borisov also provided other fixes as well as added ARC-emulation support necessary for NEC PC-98 series.
+Hermès Bélusca-Maïto also fixed several old bugs, notably a corruption in the console emulation (CONSRV) layer's memory when converting input events back to ANSI and a bug in HAL where the system lead into an assertion failure sporadically.
```

Comment:
```suggestion
Stanislav Motylkov still continues his work on Xbox boot support ranging from fixes patches and other updates brought by him. This is mainly the platform bring up, refactoring and abstraction of existing low-level code.
Hermès Bélusca-Maïto also fixed several old bugs, notably a corruption in the console emulation (CONSRV) layer's memory when converting input events back to ANSI and a bug in HAL where the system lead into an assertion failure sporadically.
```

Reply:
Well, does the XBox **really** boot to desktop now binarymaster?

Reply:
It does not without additional patches.

Reply:
Well, imho it would have been more wise then to **not** advertize that!

Reply:
PC-98 port does not boot without additional patches either.

Reply:
maybe but pc-98 wasn't already highlighted in last 0.4.13, unlike XBox https://reactos.org/project-news/reactos-0413-released/

Reply:
I just restored what Victor accidentally removed in one of suggestions.

Reply:
I am not entirely sure whether that was really removed accidentally.